src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(String color,double side) {
```

Comment:
Don't pass expressions directly into the constructor. Instead, assign the expression to a variable for better readability and then pass the variable.